Extensions/Tracing-Zipkin.md

```diff
@@ -42,14 +42,12 @@ This metadata type is intended to be used per stream, and not per connection nor
 ```
 
 * **Flags**: (8 bits)
+  * (**B**)ody Set: Tracing payload is set. 
+  * (**D**)ebug: Tracing payload should be force traced.
+  * (**S**)ample: Tracing payload should be accepted for tracing. (Ignored when D flag is set.)
+  * (**R**)eject: Tracing payload should not be sampled. (Ignored when S flag or D flag is set.)
```

Comment:
I see the reject notion is sticking with you vs sampling set. If you do that, I think reject is still a bad word to use in flags used for different things. We never use it in practice, we only used 'Reject' in the README to choose a word besides Sampled, and in that case we pair it with Accept. Reject doesn't pair with Sampled and is hence more jargon.

```suggestion
  * (**N**)ot Sampled: Tracing payload should not be sampled. (Ignored when S flag or D flag is set.)
```

I looked and in practice some say unsampled a little more say not sampled.

I think that this is important enough to pick a good name and not blindly follow the text description in the README as I've mentioned that was not trying to pick flag names and if anything we should pick names that are similar to ones in use to cause least confusion, right?

Once we settle on this point I'd like to ping some other zipkin folks as this could well be inherited into our spec.